# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1692346335 -10800
# Parent  6b1bab44ad852183aa679fc3587a2ffc0a4e57e7
Remove CSS form bug 1846611, 1848834 and 1848604 since it doesn't work.

diff --git a/mail/themes/shared/mail/about3Pane.css b/mail/themes/shared/mail/about3Pane.css
--- a/mail/themes/shared/mail/about3Pane.css
+++ b/mail/themes/shared/mail/about3Pane.css
@@ -592,16 +592,17 @@ menuitem.server[data-server-type="rss"] 
   background-color: var(--folderpane-unread-new-count-background);
 }
 
 #folderTree:focus-within li.selected > .container > .unread-count {
   background-color: var(--folderpane-unread-count-text);
   color: var(--treeitem-background-active);
 }
 
+/* Not working, bug 1846611
 .unread.children.collapsed > .container > .unread-count {
   color: var(--folderpane-unread-count-background);
   background-color: transparent;
   border: 1px solid var(--folderpane-unread-count-background);
   padding-block: 0;
 }
 
 .new-messages.unread.children.collapsed > .container > .unread-count {
@@ -609,16 +610,17 @@ menuitem.server[data-server-type="rss"] 
   border-color: var(--folderpane-unread-new-count-background);
 }
 
 #folderTree:focus-within li.selected.unread.children.collapsed > .container > .unread-count {
   color: var(--folderpane-unread-count-text);
   background-color: transparent;
   border-color: var(--folderpane-unread-count-text);
 }
+*/
 
 .folder-size {
   font-size: .8rem;
   font-weight: bold;
   color: var(--layout-color-3);
   margin-inline-end: 6px;
 }
 
diff --git a/mail/themes/shared/mail/threadPane.css b/mail/themes/shared/mail/threadPane.css
--- a/mail/themes/shared/mail/threadPane.css
+++ b/mail/themes/shared/mail/threadPane.css
@@ -105,38 +105,42 @@
   min-height: auto;
   min-width: auto;
   padding: 0;
   margin: 0;
   background-color: transparent;
   border: none;
 }
 
+/* Not working, bug 1848604
 #sizeCol,
 #unreadCol,
 #totalCol {
   min-width: 4ch;
 }
 
 #sizeColButton,
 .sizecol-column,
 #unreadColButton,
 .unreadcol-column,
 #totalColButton,
 .totalcol-column {
   text-align: end;
 }
+*/
 
 /* Add on the end aligned columns a padding when they are at the end of the row. */
+/* Not working, bug 1848834
 .sizecol-column[colspan="2"],
 .unreadcol-column[colspan="2"],
 .totalcol-column[colspan="2"] {
-  /* 25px is the width of the column picker. */
+  /* 25px is the width of the column picker.
   padding-inline-end: calc(25px + env(scrollbar-inline-size));
 }
+*/
 
 
 [is="tree-view-table-body"]:focus > .selected button,
 [is="tree-view-table-body"]:focus-within > .selected button,
 [is="tree-view-table-body"] > .selected:focus-within button {
   color: currentColor;
 }
 
